README.md

```diff
@@ -178,27 +177,34 @@ has any of these `Kind`s listed as an `OwnerRef`, that pod will not be considere
 pods created by Deployments are considered for eviction by this strategy. The `excludeOwnerKinds` parameter
 should include `ReplicaSet` to have pods created by Deployments excluded.
 
+This is a Balance plugin.
```

Comment:
plugin -> extension point
The same comment applies for the remaining occurrences. Worth mentioning a list of extension points each plugin implements in a separate table maybe?

Reply:
I removed these phrases and included a table. Extension point is wording that would prefer to avoid in a user facing readme, but I think it is fine in the table like that